src/System.IO.FileSystem/tests/Directory/Exists.cs

```diff
@@ -257,58 +250,49 @@ public void DoesCaseSensitiveComparisons()
             Assert.False(Exists(testDir.FullName.ToLowerInvariant()));
         }
 
-        [ConditionalFact(nameof(UsingNewNormalization))]
+        [ConditionalTheory(nameof(UsingNewNormalization)),
+            MemberData(nameof(SimpleWhiteSpace))]
         [ActiveIssue(20117, TargetFrameworkMonikers.Uap)]
         [PlatformSpecific(TestPlatforms.Windows)] // In Windows, trailing whitespace in a path is trimmed appropriately
-        public void TrailingWhitespaceExistence()
+        public void TrailingWhitespaceExistence(string component)
         {
             // This test relies on \\?\ support
 
             DirectoryInfo testDir = Directory.CreateDirectory(GetTestFilePath());
-            Assert.All(IOInputs.GetWhiteSpace(), (component) =>
-            {
-                string path = testDir.FullName + component;
-                Assert.True(Exists(path), path); // string concat in case Path.Combine() trims whitespace before Exists gets to it
-                Assert.False(Exists(IOInputs.ExtendedPrefix + path), path);
-            });
 
-            Assert.All(IOInputs.GetSimpleWhiteSpace(), (component) =>
-            {
-                string path = GetTestFilePath(memberName: "Extended") + component;
-                testDir = Directory.CreateDirectory(IOInputs.ExtendedPrefix + path);
-                Assert.False(Exists(path), path);
-                Assert.True(Exists(testDir.FullName));
-            });
+            string path = testDir.FullName + component;
+            Assert.True(Exists(path), path); // string concat in case Path.Combine() trims whitespace before Exists gets to it
+            Assert.False(Exists(IOInputs.ExtendedPrefix + path), path);
```

Comment:
The old code had the above three lines running only on the `IOInputs.GetWhiteSpace` components and here we have them running on the `IOInputs.GetSimpleWhiteSpace` components instead.

Do we have a test running the lines on the `IOInputs.GetWhiteSpace` components as well?